ui/app/templates/components/console/log-list.hbs

```diff
@@ -1,8 +1,10 @@
 <div class="console-ui-output has-copy-button">
-  <pre>Keys
+  <pre>
+    Keys
```

Comment:
aligns list items (see in screen shot first key is too far left)
<img width="484" alt="Screenshot 2023-04-10 at 10 34 00 AM" src="https://user-images.githubusercontent.com/68122737/230957963-6c33632b-7723-425e-84ed-53c97bbc2d0f.png">